src/actions/GraphDataActions.ts

```diff
@@ -12,7 +14,7 @@ export const GraphDataActions = {
   getGraphDataStart: createAction(GraphDataActionKeys.GET_GRAPH_DATA_START),
   getGraphDataSuccess: createAction(
     GraphDataActionKeys.GET_GRAPH_DATA_SUCCESS,
-    resolve => (timestamp: number, graphDuration: number, graphData: any) =>
+    resolve => (timestamp: TimeInMilliseconds, graphDuration: DurationInSeconds, graphData: GraphElements) =>
```

Comment:
I think the timestamp is TimeInSeconds, please verify.

Reply:
You're right!!!

https://github.com/kiali/kiali/blob/master/graph/cytoscape/cytoscape.go#L149

Reply:
I changed to TimeInSeconds!